templates/includes/sidebar.html

```diff
@@ -1,6 +1,13 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><form action="{% url 'logout' %}" method="post">{% csrf_token %}Logout</form></li>
+  {% else %}
+    <li><form action="{% url 'login' %}" method="post">{% csrf_token %}Login</form></li>
```

Comment:
The login form is missing a submit button. Add a `<button type='submit'>Login</button>` inside the form to allow users to submit the login request.